### PR TITLE
Updated changes

### PR DESCRIPTION
**Pull request by sentryadam0000345**
Got .nvmrc file updated to latest version of the demo to use node 14.2.0.  I also made a similar update to the Dockerfile as well.  I also changed the port in the docker-compose.yaml from using the internal Docker container port of 1234 to 3005.  This will make it easier to access the demo page once the docker container is running.

### CODE REVIEW

**Comment by thinkocapo** [comment]
@sentryadam0000345 

Can you put a link to a successful Sentry issue that was created as a result of this? for that DSN key you used

Can you update the README with mention of your Node, Docker or other dependencies that are relevant here?

**Comment by sentryadam0000345** [comment]
Ok

File: public/index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
 
   <script>
     Sentry.init({
-      dsn: 'https://f3e1b754a04b4dc1be1b54454ec6d238@sentry.io/1356467',
+      dsn: 'https://a843d07bf5974b14937a1314f49ad4bb@o87286.ingest.sentry.io/5230364',
```
Comment (thinkocapo) [inline public/index.html:30]: test

File: docker-compose.yaml [hunk 2]
```diff
@@ -6,6 +6,6 @@ services:
       dockerfile: Dockerfile
     restart: always
     ports:
-     - "3005:1234" # React app will be served on localhost:3005
+     - "3005:3005" # React app will be served on localhost:3005
```
[the remark this answers is not in the document]
Comment (sentryadam0000345) [reply docker-compose.yaml:9]: @thinkocapo Correct it was working but for clarity, I think it is best to have the ports the same.

**Pull request merged**